tokens.toml

```diff
@@ -0,0 +1,10 @@
+# Token accounts with their validity predicates
+
+[token.NAAN]
+denom = 6
+
+[token.NAM.masp_params]
+max_reward_rate = "0.0"
+kd_gain_nom = "0.1"
+kp_gain_nom = "0.1"
+locked_ratio_target = "0.0"
```

Comment:
I think it is fine to set this to the nominal value of "0.1". Keeping `max_reward_rate` at "0.0" should alone ensure that the MASP inflation remains 0. This will just be one less thing that we may possibly forget to change later when enabling inflation.

Reply:
Do you mean `locked_ratio_target` @brentstone ?

Reply:
@bengtlofgren yes

Reply:
I believe that the name of this parameter will change with @brentstone's PR